mw-webapp/src/utils/KeyboardSymbols.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Keyboard symbols
+ */
+export enum KeyboardSymbols {
```

Comment:
we shuld differentiate event.keycode, key.key, key.code etc. Specify naming and comment, currently it is unclear

Reply:
Rebamed and added comment